vuu-ui/packages/vuu-layout/test/layout-persistence/RemoteLayoutPersistenceManager.test.ts

```diff
@@ -0,0 +1,311 @@
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import { RemoteLayoutPersistenceManager } from "../../src/layout-persistence/RemoteLayoutPersistenceManager";
+import { LayoutMetadata } from "@finos/vuu-shell";
+import { LayoutJSON } from "../../src/layout-reducer";
+import { v4 as uuidv4 } from "uuid";
+import { expectPromiseRejectsWithError } from "./utils";
+
+const persistence = new RemoteLayoutPersistenceManager();
+const mockFetch = vi.fn();
+
+global.fetch = mockFetch;
+
+const metadata: LayoutMetadata[] = [
+  {
+    id: "0001",
+    name: "layout 1",
+    group: "group 1",
+    screenshot: "",
+    user: "username",
+    created: "01.01.2000",
+  },
+];
+
+const metadataToAdd: Omit<LayoutMetadata, "id" | "created"> = {
+  name: "layout 1",
+  group: "group 1",
+  screenshot: "",
+  user: "username",
+};
+
+const layout: LayoutJSON = {
+  type: "",
+};
```

Comment:
For the screenshots and type, I would put some dummy text in rather than empty strings. It might just safeguard against potential edge cases.

Reply:
good point. I will make the change

Reply:
done!

Reply:
I think you've missed one for screenshot on line 18.

Reply:
done